src/css/modules/_Contact.scss

```diff
@@ -2,15 +2,16 @@
   border: 1px solid $coa-color-purple-lighter;
   border-left: none;
   border-right: none;
-  padding: $coa-spacing-large 0;
+  padding: $coa-spacing-xlarge 0 $coa-spacing-large;
 }
 
-.coa-ContactDetails__item {
+.coa-ContactItem {
```

Comment:
Renamed to ContactItem, so Contact subcomponents like hours, phone, email weren't required to be nested under ContactDetails. (since they now show up under ContactMap too)